zoe-cli/test/com/adevinta/oss/zoe/cli/testcontainers.kt

```diff
@@ -0,0 +1,57 @@
+package com.adevinta.oss.zoe.cli
+
+import com.adevinta.oss.zoe.core.utils.logger
+import org.testcontainers.containers.GenericContainer
+import org.testcontainers.containers.KafkaContainer
+import org.testcontainers.containers.Network
+import org.testcontainers.containers.output.Slf4jLogConsumer
+
+private const val kafkaVersion = "5.5.1"
+
+/**
+ * Context which holds the started containers for easy access in application setup customizers.
+ */
+object TestcontainersContext {
+    private lateinit var kafkaContainer: KafkaContainer
+    val kafka get() = kafkaContainer
+
+    private lateinit var schemaRegistryContainer: SchemaRegistryContainer
+    val schemaRegistry get() = schemaRegistryContainer
+
+    fun start() {
+        val network = Network.newNetwork()
+        val logConsumer = Slf4jLogConsumer(logger)
+
+        kafkaContainer = KafkaContainer(kafkaVersion)
+            .withNetwork(network)
+            .withLogConsumer(logConsumer)
+            .apply { start() }
+
+        schemaRegistryContainer = SchemaRegistryContainer(kafka)
+            .withLogConsumer(logConsumer)
+            .apply { start() }
+    }
+
+    fun stop() {
+        schemaRegistryContainer.stop()
+        kafkaContainer.stop()
+    }
+}
+
+class SchemaRegistryContainer(kafka: KafkaContainer) :
```

Comment:
can you inject only the `network` here instead of the whole `KafkaContainer` to make the intent clearer?